### PR TITLE
maxSubscriptionsPerTopic support cross multiple clusters

### PR DESCRIPTION
**Pull request by suiyuzeng**
Master Issue: #13444
### Motivation
maxSubscriptionsPerTopic support cross multiple clusters

###  Documentation
  
- [x] `no-need-doc`

### CODE REVIEW

File: pulsar-client-tools-test/src/test/java/org/apache/pulsar/admin/cli/PulsarAdminToolTest.java [hunk 1]
```diff
@@ -1187,6 +1195,13 @@ public void topicPolicies() throws Exception {
         verify(mockGlobalTopicsPolicies).getSubscriptionDispatchRate("persistent://myprop/clust/ns1/ds1", false);
         cmdTopics.run(split("remove-subscription-dispatch-rate persistent://myprop/clust/ns1/ds1 -g"));
         verify(mockGlobalTopicsPolicies).removeSubscriptionDispatchRate("persistent://myprop/clust/ns1/ds1");
+
+        cmdTopics.run(split("get-max-subscriptions-per-topic persistent://myprop/clust/ns1/ds1 -g"));
+        verify(mockTopicsPolicies).getMaxSubscriptionsPerTopic("persistent://myprop/clust/ns1/ds1");
```
Comment (Jason918) [inline pulsar-client-tools-test/src/test/java/org/apache/pulsar/admin/cli/PulsarAdminToolTest.java:1200]: should  use mockGlobalTopicsPolicies  here

Comment (suiyuzeng) [reply pulsar-client-tools-test/src/test/java/org/apache/pulsar/admin/cli/PulsarAdminToolTest.java]: fixded

**Review by Jason918 — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**